pkg/analysis/passes/discoverability/discoverability.go

```diff
@@ -0,0 +1,42 @@
+package discoverability
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+)
+
+var (
+	emptyDescription = &analysis.Rule{Name: "empty-description", Severity: analysis.Warning}
+	emptyKeywords    = &analysis.Rule{Name: "empty-keywords", Severity: analysis.Warning}
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "discoverability",
+	Requires: []*analysis.Analyzer{},
```

Comment:
You are reading the result of `metadata.Analyzer` in `run` (like 23), so that analyzer has to run before this one:

```suggestion
	Requires: []*analysis.Analyzer{metadata.Analyzer},
```